app/controllers/staff/organisations_controller.rb

```diff
@@ -3,24 +3,23 @@
 class Staff::OrganisationsController < Staff::BaseController
   def index
     @organisations = policy_scope(Organisation)
-    authorize @organisations
   end
 
   def show
-    organisation = policy_scope(Organisation).find(params[:id])
+    organisation = Organisation.find(id)
     authorize organisation
 
     @organisation_presenter = OrganisationPresenter.new(organisation)
-    @funds = Fund.includes(:organisation).where(organisation: organisation)
+    @funds = policy_scope(Fund).includes(:organisation).where(organisation: organisation)
```

Comment:
Given that funds are listed on the organisation show page, we need to make sure we only show funds for a user who's a fund manager. Delivery partners shouldn't see any.